login-workflow/example/src/app/pages/auth/auth.component.ts

```diff
@@ -28,6 +28,10 @@ export class AuthComponent {
         pxbAuthConfig.backgroundImage = 'assets/images/background.svg';
         pxbAuthConfig.allowDebugMode = true;
         pxbAuthConfig.showSelfRegistration = false;
+        pxbAuthConfig.passwordRequirements.push({
+            regex: /^((?!password).)*$/,
+            description: 'Does not contain "password"',
+        });
```

Comment:
Does this mean the user can only extend the default requirements? What if they want to pass in their own completely unique set?

Reply:
They can push to the array or completely replace it. We might want to document that somewhere.

Reply:
I'll add a blurb in the README to explain better 👍